File: Modul-2/soal-2/PRAK202_2410817210017_OrlandoSugian.c
#include<stdio.h>
int main()
{
    float a,b;
    printf("Masukkan Nilai Pertama : ");
    scanf("%f", &a);
    printf("Masukkan Nilai Kedua : ");
    scanf("%f", &b);
    if(a < 1)
    printf("Hasil dari penjumlahan nilai pertama \"%.2f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"", a, b, a+b);
    else 
    printf("Hasil dari penjumlahan nilai pertama \"%.0f\" dan nilai kedua \"%.1f\" adalah \"%.2f\"", a, b, a+b);
    return 0;
}

File: Modul-2/soal-2/PRAK202_2410817210017_OrlandoSugian.py
nilai1 = float(input("Masukkan Nilai Pertama : "))
nilai2 = float(input("Masukkan Nilai Kedua : "))
hasil = nilai1 + nilai2
if (nilai1 < 1) :
    print(f"Hasil dari penjumlahan nilai pertama \"{nilai1:.2f}\" dan nilai kedua \"{nilai2:.1f}\" adalah \"{hasil:.2f}\"")
else : 
    print(f"Hasil dari penjumlahan nilai pertama \"{nilai1:.0f}\" dan nilai kedua \"{nilai2:.1f}\" adalah \"{hasil:.2f}\"")    